tests/server/master_server/lock_manager_unit_test.cc

```diff
@@ -4,33 +4,38 @@
 #include "gtest/gtest.h"
 #include "src/server/master_server/lock_manager.h"
 
+using namespace gfs::server;
+
 class LockManagerUnitTest : public ::testing::Test {
  protected:
   void SetUp() override {
-    lockManager_ = gfs::server::LockManager::GetInstance();
+    lockManager_ = LockManager::GetInstance();
   }
 
-  gfs::server::LockManager* lockManager_;
+  LockManager* lockManager_;
 };
 
+// Add lock step by step and make sure the exist
 TEST_F(LockManagerUnitTest, AddLock) {
   EXPECT_EQ(lockManager_->Exist("/foo"), false);
   // Successfully add a lock for /foo
-  auto foo(lockManager_->AddLockIfNonExist("/foo"));
-  EXPECT_NE(foo, nullptr);
+  auto fooRes(lockManager_->CreateLock("/foo"));
+  EXPECT_EQ(fooRes.ok(), true);
```

Comment:
Use `EXPECT_TRUE` and `EXPECT_FALSE`  for all other places including here

Reply:
All converted